sdk/dotnet/Secret.cs

```diff
@@ -81,11 +85,21 @@ public static Secret Get(string name, Input<string> id, SecretState? state = nul
 
     public sealed class SecretArgs : global::Pulumi.ResourceArgs
     {
+        [Input("data", required: true)]
+        private Input<string>? _data;
+
         /// <summary>
         /// Base64-url-safe-encoded secret data
         /// </summary>
-        [Input("data", required: true)]
-        public Input<string> Data { get; set; } = null!;
+        public Input<string>? Data
```

Comment:
I think a lot of the diffs here come from upgrading the reference to codegen so we get some updates to how C# is laid out. Perhaps we could check those in in a separate PR... well nevermind this one is close but next time there's a situation like this, let's maybe check in the updates separately first, helps to review .